src/code/zurumode.c

```diff
@@ -185,7 +186,7 @@ void zurumode_callback(void* p) {
         zerucheck_init(&zuruKeyCheck);
         APPNMI_ZURUMODE_FLP();
         zurumode_update();
-        if ((mgr->pads[0].button) & 0x80) {
+        if ((mgr->pads[0].button) & RESET_BUTTON) {
```

Comment:
```suggestion
        if (CHECK_BTN_ALL(mgr->pads[0].button, RESET_BUTTON)) {
```

Reply:
Does that still match? 🤔 The macro looks very different than what is here

Reply:
Macro doesn't match, so it can't be used here

Reply:
```suggestion
        if (mgr->pads[0].button & RESET_BUTTON) {
```
Ah dang. Than I would suggest just removing the extra parens.